src/index.js

```diff
@@ -109,4 +109,5 @@ import WizardLayout from "./WizardLayout";
 export { WizardLayout };
 
 import FloatingButton from "./FloatingButton";
-export { FloatingButton };
+import { ColorGroup, PositionX, PositionY } from "./FloatingButton";
+export { ColorGroup, FloatingButton, PositionX, PositionY };
```

Comment:
Given that `ColorGroup`, `PositionX`, and `PositionY` are pretty generic names that other components might some day export, let's actually attach them as a static fields to the component itself rather than exporting them separately:

https://github.com/Clever/components/blob/40c581b19d1d8a15f3032123e7699bf6427ff10d/src/TopBar/index.tsx#L22-L25

```ts
export default class FloatingButton extends React.PureComponent {
  static ColorGroup = ColorGroup;
  static PositionX = PositionX;
  static PositionY = PositionY;
  ...
}
```

Client-code can then do something like this:

```jsx
import { FloatingButton } from "clever-components";

<FloatingButton
  colorGroup={FloatingButton.ColorGroup.GREEN}
/>
```

Reply:
I actually expect ColorGroup to be more generic, but on board for the others

Reply:
for now I'll make it a class field though